content/markdown/03-plotting/03-02-iterative-plotting.rmd

```diff
@@ -0,0 +1,294 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Iterative Plotting for Easy and Aesthetic Figures
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+```{r, echo=FALSE}
+options(tidyverse.quiet = TRUE)
+```
+
+When making a plot, typically the best way to go about it is to iteratively make a version of the plot, observe the outcome, and then make changes in an iterative fashion. This prevents us from making mistakes in a long set of code that we then can't diagnose the problem with, and further, helps us catch mistakes (typos, colour problems, etc) as we only have to take in a few new components each time, and not a whole pile. 
```

Comment:
I might suggest the following tweaks to this paragraph: "When making a plot, it's best to build your plot iteratively: make a version of the plot, observe the outcome, and then make changes or add an element and replot. This helps us catch mistakes as they occur (typos, colour problems, etc) and prevents small problems from creating big, difficult to diagnose problems in a long set of code."